nixos/modules/services/misc/nscd-sssd.conf

```diff
@@ -0,0 +1,15 @@
+server-user             nscd
+threads                 1
+paranoia                no
+debug-level             0
+
+enable-cache            passwd          no
+enable-cache            group           no
```

Comment:
I don't get any errors and I can see SSSD processing the request when tracing is switched on - please see above.